### DISCUSSION TITLE
I can't hash user emails

### DISCUSSION THREAD

**Discussion by bright258** [supabase/supabase#15681] [Feature Requests]
# Improve documentation

## Describe the problem

I will love to hash user emails incase of a hack or raid on my platform.

## Describe the improvement

The should be a setting or cli command for hashing emails of authenticated users.

## Additional context
(https://supabase.com/dashboard/project/hwxfpyrzqxkgqoomjzns/auth/users)

**Comment by GaryAustin1** [comment]
Not sure why you flagged this as documentation.  This sounds like a feature request.
